util/tabComplete.js

```diff
@@ -13,6 +13,11 @@ const command = (partial) => {
   return commands.filter(cmd => cmd.startsWith(partial))
 }
 
+const emoji = (partial) => {
+  const emojiList = Object.keys(emojis.emoji)
```

Comment:
actually how about mapping the list ```Object.keys(emojis.emoji).map(e => `:${e}:`)``` so that the list has the colons already. then you don't have to return on line 32 and can use the global return at the bottom